core/src/test/java/io/parsingdata/metal/expression/value/ExpandTest.java

```diff
@@ -99,7 +99,7 @@ public void expandList() {
         }
     }
 
-    private static ParseValue createParseValue(String name, int value) {
+    public static ParseValue createParseValue(String name, int value) {
```

Comment:
I tend to not pay much attention to these things in test code, but you have a point. I'll make an issue so collect all these _dummy_ values in a static `TestData` class or something like that.